### ISSUE TITLE
Player accessibility

### ISSUE THREAD

**Issue by SimonRothUCF** [ucfopen/guess-the-phrase-materia-widget#33]
- The "Start" button at the beginning and the "Finish" button at the end both cannot be tabbed into and are not otherwise accessible from the keyboard. They should be made tab-reachable and/or the player should be able to press Enter to advance like they can between questions. Other than this, the game can be completed using the keyboard.
- The widget does not appear to be screen reader accessible at all (using Voice Over). I did not find any elements that could be tabbed into.
- The question text needs to be screen reader accessible.
- The number of available spaces and found letters should be screen reader accessible.
- The number of incorrect guesses and how many mistakes the player has left should be screen reader accessible.
- There should be an audio indication for whether the player gets a question right or wrong.
- It might be nice to add screen reader support for visual cues, like when the host changes his expression.
